packages/api/tests/endpoints/executions.js

```diff
@@ -1,95 +0,0 @@
-'use strict';
```

Comment:
I assume that the plan is to move these tests, not delete them, correct?

Reply:
Actually these are all duplicate tests. These, and more, are all in here already https://github.com/nasa/cumulus/blob/master/packages/api/tests/serial/endpoints/executions.js